COPYING.Apache-2.0

```diff
@@ -201,4 +200,3 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
We should get rid of these set of instructions (from line 177)

Reply:
Based on http://www.apache.org/dev/licensing-howto.html#simple I think it would be safe to keep it in place.